spyder/plugins/shortcuts/plugin.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2009- Spyder Project Contributors
+#
+# Distributed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+# -----------------------------------------------------------------------------
+
+"""
+Shortcuts Plugin.
+"""
+
+# Local imports
+from spyder.api.plugins import SpyderPluginV2
+from spyder.api.translations import get_translation
+from spyder.plugins.shortcuts.confpage import ShortcutsConfigPage
+from spyder.plugins.shortcuts.widgets.summary import ShortcutsSummaryDialog
+
+# Localization
+_ = get_translation('spyder')
+
+
+# --- Plugin
+# ----------------------------------------------------------------------------
+class Shortcuts(SpyderPluginV2):
+    """
+    Shortcuts Plugin.
+    """
+
+    NAME = 'shortcuts'
+    # TODO: Fix requires to reflect the desired order in the preferences

```

Comment:
What does this TODO mean? I think it doesn't apply in this case, right?

Reply:
It applies if the order in Preferences wants to be in a certain way.
That is why is left there as TODO (to fix in a separate PR when all API plugins are migrated).

Also at that point, it will be worth discussing if the order in the settings should be tied to the order of registry means order of appearing in the dialog. Maybe Alphabetically makes more sense.

Something the UI/UX team can ponder on.